arches_her/migrations/0003_add_bng_search_view.py

```diff
@@ -0,0 +1,73 @@
+from django.db import migrations, models
+from django.utils.translation import gettext as _
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("arches_her", "0002_move_pkg_sql"),
+        ("models", "10804_core_search_filters"),        
```

Comment:
I think this `10804_core_search_filters ` dependency may cause problems because it is already a dependency of another arches migration, `11117_add_bulk_concept_editor`. 

I think it should probably be changed to the last migration in arches 7.6 (`11499_add_editlog_resourceinstance_idx`) and moved to be the first dependency of `0002_move_pkg_sql.py`. That way you finish Arches 7.6 migrations, and then resume the arches_her migrations at `0002_move_pkg_sql `.

In other words, make sure Arches updates to `11499_add_editlog_resourceinstance_idx` before `0002_move_pkg_sql` runs, then run `0003_add_bng_search_view`.